scripts/test_gui_components.py

```
@@ -0,0 +1,164 @@
+        if contact:
+            print(f"✅ Contact retrieved: {contact}")
+        else:
+            print("❌ Failed to retrieve contact")
+            return False
+        
+        # Test message processor
+        processor = MessageProcessor(db_manager)
+        print("✅ Message processor created")
+        
+        # Test export manager
+        export_manager = ExportManager(db_manager, processor)
```

Ruff (0.14.0)

Local variable `export_manager` is assigned to but never used

Remove assignment to unused variable `export_manager`

(F841)